libs/@hashintel/design-system/src/configure-block-button.tsx

```diff
@@ -0,0 +1,55 @@
+import { buttonClasses, ButtonProps } from "@mui/material";
+import { FunctionComponent } from "react";
+
+import { Button } from "./button";
+import { ChevronRightIcon } from "./icons/chevron-right";
+import { GearIcon } from "./icons/gear";
+
+export type ConfigureBlockButtonProps = {
+  expanded?: boolean;
+} & ButtonProps;
+
+export const ConfigureBlockButton: FunctionComponent<
+  ConfigureBlockButtonProps
+> = ({ expanded = false, sx, ...props }) => {
+  return (
+    <Button
+      {...props}
+      variant="tertiary_quiet"
+      size="xs"
+      startIcon={<GearIcon />}
+      endIcon={<ChevronRightIcon sx={{ fontSize: "12px !important" }} />}
+      sx={[
+        ({ palette, transitions }) => ({
+          p: 0,
+          gap: 1,
+          minHeight: 0,
+          fontSize: 15,
+          lineHeight: 1,
+          letterSpacing: -0.02,
+          whiteSpace: "nowrap",
+          background: "none",
+          color: `${palette.gray[50]}`,
+          ":hover": {
+            background: "none",
+            color: palette.gray[60],
+            [`& .${buttonClasses.startIcon}, .${buttonClasses.endIcon}`]: {
+              color: palette.gray[50],
+            },
+          },
+          [`.${buttonClasses.startIcon}, .${buttonClasses.endIcon}`]: {
+            mx: 0,
+            transition: transitions.create(["color", "transform"]),
+            color: palette.gray[40],
+          },
+          [`.${buttonClasses.endIcon}`]: {
+            transform: `rotate(${expanded ? 90 : 0}deg)`,
+          },
+          ...(Array.isArray(sx) ? sx : [sx]),
+        }),
+      ]}
+    >
+      Configure block
```

Comment:
```suggestion
      Configure block
```

@luisbettencourt Sorry for flip-flopping on this, but can we change this to read "Block settings"?

"Configure" is an action word, but it's quite possible the user will just want to _inspect_ the contents, and not actually modify it. "Block settings" makes it more neutral. Thank you/sorry!

We should probably rename the file, consts, etc too.

Reply:
LGTM, thanks. Over to code review

Reply:
Belatedly now realizing... hadn't we originally said it would only show as a dropdown on mobile, @luisbettencourt?

If there are _lots_ of items in block settings we could keep the dropdown on desktop, but provided they all fit on one row, we should just show them to make the options more discoverable, and save people the extra click.

Visibly highlighting the underlying services in use within any given block is almost certainly going to be encouraged as general best practice, so let's do our best to abide by it here in anticipation.

Reply:
The `Block settings` button is being displayed when the block container width is less than 800px. 
However, blocks in the WP editor have a max-width of 650px, which means it will always be displayed there. 

![responsiveness](https://user-images.githubusercontent.com/37453800/226366942-fa001377-f0a1-4e69-b658-70a1f7c06c3c.gif)

Reply:
Is there anything we can do make sure the `Block settings` button is _not_ always hidden in WordPress?

We want it to be visible as much of the time as possible and _only_ collapsed when it would otherwise bleed onto multiple lines.

It seems in the above example it is breaking overly-early. Instead, it should break dependent on the width of its contained content.

Likewise, it seems like we break for the "Enter an address" copy in the input overly prematurely.

Reply:
Reduced the breakpoint to `620px` (which makes it not visible by default in WP):

![responsiveness](https://user-images.githubusercontent.com/37453800/226778424-fb802afd-3964-4b89-ada1-541628e16bc0.gif)

Also made it so the input placeholder changes for a smaller breakpoint.

Reply:
Looks better, thank you. You mean the full bar _is now_ visible by default in WP, though, or it's still not?